test/workers/generic_worker_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+class GenericWorkerTest < ActiveSupport::TestCase
+  def setup
+    super
```

Comment:
Tests will run faster if you don't run everything that `super` runs. I suggest you start with empty `setup` and `teardown` methods and then add only the steps you actually need:

```
def setup
end

def teardown
end
```